python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -543,7 +536,7 @@ def _run_iteration_impl(
             for sensor, repo in eligible_sensors_and_repos:
```

Comment:
confirming understanding - this does not let any new sensors through because we did a more correct sensor_type check above based on the workspace snapshot to determine  eligible_sensors_and_repos

Reply:
exactly yep!